src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/BlobAuditingTests.cs

```diff
@@ -0,0 +1,180 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ScenarioTest.SqlTests;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+using Xunit.Abstractions;
+using RestTestFramework = Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+
+namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
+{
+    public class BlobAuditingTests : SqlTestsBase
+    {
+        protected override void SetupManagementClients(RestTestFramework.MockContext context)
+        {
+            var sqlClient = GetSqlClient(context);
+            var sqlLegacyClient = GetLegacySqlClient();
+            var storageClient = GetStorageClient();
+            var storageV2Client = GetStorageV2Client();
+            var resourcesClient = GetResourcesClient();
```

Comment:
@ranisha2 @jaredmoo @akromm  FYI, yesterday we [made some changes to move dependencies into a common project](https://github.com/Azure/azure-powershell/issues/3437), as well as use generated libraries where we could, which included the Resources library. As a result, we had to make some changes to the Sql test classes so that they used the "new" `ResourcesClient` ([here is an example](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AuditingTests.cs#L32) and the corresponding [method in `SqlTestBase`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/SqlTestsBase.cs#L150)). The build is currently failing for the tests that you have added since it cannot find this `ResourcesClient`. Once you add it to the constructor of this test class, and add it to the `SetupSomeOfManagementClients` method, the tests should pass.